src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -306,58 +305,26 @@ private Future saveSnapshots(List<JsonObject> snapshots, OkapiConnectionParams p
     Snapshot snapshot = new Snapshot()
       .withJobExecutionId(jobExecution.getId())
       .withStatus(Snapshot.Status.fromValue(jobExecution.getStatus().name()));
-    RestUtil.doRequest(params, url, HttpMethod.PUT, snapshot).setHandler(updateResult -> {
-      if (isValidAsyncResult(updateResult, future)) {
-        future.complete(jobExecution);
-      } else {
-        jobExecutionDao.updateBlocking(jobExecution.getId(), jobExec -> {
-          Future<JobExecution> jobExecutionFuture = Future.future();
-          jobExec.setErrorStatus(JobExecution.ErrorStatus.SNAPSHOT_UPDATE_ERROR);
-          jobExec.setStatus(JobExecution.Status.ERROR);
-          jobExec.setUiStatus(JobExecution.UiStatus.ERROR);
-          jobExecutionFuture.complete(jobExec);
-          return jobExecutionFuture;
-        }).setHandler(jobExecutionUpdate -> {
-          String message = "Couldn't update snapshot status for jobExecution with id " + jobExecution.getId();
-          LOGGER.error(message);
-          future.fail(message);
-        });
-      }
-    });
+    RestUtil.doRequest(params, url, HttpMethod.PUT, snapshot)
+      .setHandler(updateResult -> {
+        if (RestUtil.validateAsyncResult(updateResult, future)) {
+          future.complete(jobExecution);
+        } else {
+          jobExecutionDao.updateBlocking(jobExecution.getId(), jobExec -> {
+            Future<JobExecution> jobExecutionFuture = Future.future();
+            jobExec.setErrorStatus(JobExecution.ErrorStatus.SNAPSHOT_UPDATE_ERROR);
+            jobExec.setStatus(JobExecution.Status.ERROR);
+            jobExec.setUiStatus(JobExecution.UiStatus.ERROR);
+            jobExecutionFuture.complete(jobExec);
+            return jobExecutionFuture;
+          }).setHandler(jobExecutionUpdate -> {
+            String message = "Couldn't update snapshot status for jobExecution with id " + jobExecution.getId();
```

Comment:
Can not update ...

Reply:
"Couldn't" is fine, but if you prefer using present simple, then "cannot" is much more common